src/components/AddQuestion.js

```diff
@@ -0,0 +1,64 @@
+import { Button, HStack, Input, useToast } from '@chakra-ui/react';
+import { useEffect, useState } from 'react';
+import { useAuthContext } from './AuthContextProvider';
+
+const AddQuestion = ({ addQuestion }) => {
+  const { questions, setQuestions } = useAuthContext();
```

Comment:
Questions data should not be stored in AuthContext - you can create a new context for that - QuestionContext